src/EPR.ProducerContentValidation.Application/Exceptions/SubmissionApiClientException.cs

```diff
@@ -1,7 +1,6 @@
 ﻿namespace EPR.ProducerContentValidation.Application.Exceptions;
 
 using System.Diagnostics.CodeAnalysis;
-using System.Runtime.Serialization;
 
 [ExcludeFromCodeCoverage]
 [Serializable]
```

Comment:
Consider removing the serializable attribute since the rest of the serialisation code has been removed.